src/ZlibSharp/ZlibSharp/MemoryZlib.cs

```diff
@@ -3,269 +3,108 @@
 // All rights reserved.
 // license: MIT, see LICENSE for more details.
 
+using System.Runtime.CompilerServices;

```

Comment:
Add this to ``Directory.Build.targets`` please as ``<Using Include="System.Runtime.CompilerServices" />`` under the ItemGroup. Also look carefully as I prefer it to be alphabetized.